package.json

```diff
@@ -53,29 +53,34 @@
     "eslint-plugin-react": "^6.4.1",
     "eslint-plugin-standard": "^3.0.1",
     "jest": "^19.0.2",
-    "less": "^2.7.2",
-    "less-loader": "^4.0.5",
-    "patternfly": "^3.27.4",
+    "node-sass": "^4.7.2",
+    "patternfly": "^3.31.0",
     "prettier": "^1.7.4",
     "raf": "^3.4.0",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
     "react-test-renderer": "^16.0.0",
+    "sass-loader": "^6.0.6",
     "semantic-release": "^6.3.6",
     "style-loader": "^0.19.0"
   },
   "peerDependencies": {
-    "patternfly": "^3.27.4",
+    "patternfly": "^3.31.0",
     "prop-types": "^15.5.10",
     "react": "^15.6.2 || ^16.0.0",
     "react-dom": "^15.6.2 || ^16.0.0"
   },
+  "sassIncludes": {
```

Comment:
What is this?

Reply:
@TheRealJon http://ianmcnally.me/blog/2016/2/4/variables-in-npm-scripts

Reply:
Yeah, the line for the node-sass command was very long and hard to read what paths were being include. This attempts to make it more readable/maintainable.

Reply:
Ahh, okay. I figured it was something like that, but had not seen this before.